kdwsdl2cpp/src/converter.cpp

```diff
@@ -93,7 +93,7 @@ class TypeCollector
     void registerDerivedClasses()
     {
         XSD::ComplexType::List complexTypes = m_allTypes.complexTypes();
```

Comment:
Make the list const so that range-for doesn't detach, or use qAsConst if the list is modified later on.